terraform/providers.tf

```diff
@@ -1,5 +1,7 @@
 provider "azurerm" {
   version = ">= 1.36.0"
+
+  features {}
```

Comment:
[Build failed when features property is empty, either set the values or remove this block](https://dev.azure.com/hmctsreform/VirtualHearings/_build/results?buildId=24434&view=logs&j=6ff741a4-39f6-5bba-3d66-dbed9abaa323&t=d130335d-d1c3-54ed-07c2-6bb419277855&l=62)